src/features/home/screen/index.tsx

```diff
@@ -94,10 +94,10 @@ const HomeScreen: React.FunctionComponent = () => {
         value={search}
         containerStyle={styles.searchBarContainer}
       />
-      {products && products.data.length !== 0 && search.length > 0 ? (
+      {products && products.length !== 0 && search.length > 0 ? (
```

Comment:
Similar to one of my comments.

```suggestion
      {products.length && search.length ? (
```